MyForage/MyForage/Models/WeatherHistory+Convinience.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  WeatherHistory+Convinience.swift
+//  MyForage
+//
+//  Created by Zachary Thacker on 1/5/21.
+//
+
+import Foundation
+import CoreData
+
+extension WeatherHistory {
+    @discardableResult convenience init(dateTime: Date = Date(),
```

Comment:
I think we don't need the default value for date here, since we don't really want the current date and time to be saved with each weather history day.

Reply:
```suggestion
    @discardableResult convenience init(dateTime: Date,
```